src/main/flight/setpoint.c

```diff
@@ -200,6 +208,10 @@ void setpointUpdate(void)
 
         SP = sp.setpoint[axis] = applyRatesCurve(axis, SP);
         DEBUG_AXIS(SETPOINT, axis, 4, SP);
+
```

Comment:
Are you confident that the boost should be done after the rate curve application?

Reply:
Since it's "Setpoint boost", I feel it's reasonable after the rate curve. Otherwise it would be more like "stick boost"? @etocii to justify.